apis/v1alpha2/shared_types.go

```diff
@@ -84,15 +84,15 @@ type ParentReference struct {
 	SectionName *SectionName `json:"sectionName,omitempty"`
 
 	// Port is the network port this Route targets. It can be interpreted
-	// differently based on the type of parent resource:
-	//
-	// * Gateway: All listeners listening on the specified port that also
-	// support this kind of Route(and select this Route). It's not
-	// recommended to set `Port` unless the networking behaviors specified
-	// in a Route must apply to a specific port as opposed to a listener(s)
-	// whose port(s) may be changed. When both Port and SectionName are
-	// specified, the name and port of the selected listener must match both
-	// specified values.
+	// differently based on the type of parent resource.
```

Comment:
This does beg the question ... "what other parent resources are supported other than Gateway?"